meshtastic/tests/test_int.py

```
@@ -7,40 +8,53 @@
 
 @pytest.mark.int
 def test_int_meshtastic_no_args():
-    """Test meshtastic without any args"""
-    return_value, out = subprocess.getstatusoutput("meshtastic")
-    assert re.match(r"usage: meshtastic", out)
-    assert return_value == 1
+    """Test meshtastic without any args."""
+    result = subprocess.run(["meshtastic"], capture_output=True, text=True, check=False)
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)